src/config-import.js

```diff
@@ -479,6 +511,24 @@ function parseIcon(filename, buf) {
   }
 }
 
+/**
+ * @param {Record<string,unknown>} obj
+ * @returns {obj is MetadataFile}
+ */
+function isValidMetadataFile(obj) {
```

Comment:
should we allow extra fields in `metadata.json` I would say since it would mean not needing to update the config builder, and we can ignore values we don't want

Reply:
Yes, I think we should allow extra fields. This seems good to me.

Reply:
How about using ajv and JSONSchema for this? We already have them as deps via mapeo-schema. Could be more work than necessary for now.

Reply:
(heh, was already doing that: MetadataFile is grabbed from @mapeo/schema)